pkg/arm/data/node-startup.sh

```diff
@@ -58,3 +58,7 @@ systemctl start ${SERVICE_TYPE}-node.service &
 # disabling rsyslog since we manage everything through journald
 systemctl disable rsyslog.service
 systemctl stop rsyslog.service
+
+#load the recommended tuned profile after node is connected to cluster
+#currently it takes <10min from beginning of arm template deployment until the end of the cluster creation process
+echo 'systemctl restart tuned' | at now + 10min
```

Comment:
This really is not up to the standard that our codebase needs to be at.  Don't do this.

Reply:
understood.